vm.go

```diff
@@ -540,6 +540,45 @@ func (vm *vm) captureStack(stack []StackFrame, ctxOffset int) []StackFrame {
 	return stack
 }
 
+// TODO(REALMC-12007): remove this temporary logs that were added to debug a prod panic
+func (vm *vm) captureStackWithLogs(stack []StackFrame, ctxOffset int) string {
+	var logs string
+
+	// Unroll the context stack
+	if vm.pc != -1 {
```

Comment:
should we flip this and return an empty string if `vm.pc == -1`? save us from having to try to enter the loop below

Reply:
yeah why not! It makes sense in this context